src/services/imu_service/imu_service.cpp

```diff
@@ -99,6 +99,14 @@ void ImuService::OnCreate() {
   ULOG_ARG_INFO(&service_id_, "IMU configured successfully");
 }
 
+bool ImuService::OnStart() {
+  // Type-sicheres Kopieren
+  memcpy(axis_remap_.data(), AxisRemap.value, AxisRemap.length * sizeof(uint8_t));
+  memcpy(axis_sign_.data(), AxisSign.value, AxisSign.length * sizeof(int8_t));
+
+  return true;
+}
+
```

Comment:
Is the copy necessary? The register values should be available in tick() as well, and I can't think of a reason why they shouldn't be used. I might be missing something though.

Reply:
Really?! Why-ever I assumed they'll go away. If not, then I could drop OnStart() completely.

Reply:
`DiffDriveService::ProcessStatusUpdate()` uses register values, so I assume it's meant that way, and since the memory is statically allocated, I can't imagine why they should vanish.

Reply:
yes it's available in tick as well